Resources/Prototypes/Catalog/Fills/Crates/armory.yml

```diff
@@ -2,7 +2,7 @@
   id: CrateArmorySMG
   parent: [ CrateWeaponSecure, BaseRestrictedContraband ]
   name: SMG crate
-  description: Contains two high-powered, semiautomatic rifles with four mags. Requires Armory access to open.
+  description: Contains two high-powered, automatic carbines with four extra magazines. Requires Armory access to open.
```

Comment:
I find it interesting that we don't just call SMGs "Submachine guns". 

Also, I'd consider changing the description to make it more obvious what model it gives: "Contains two WT550 submachine guns and four top-mounted magazines. [...]"